packages/reporting-server/setup.py

```diff
@@ -31,6 +31,7 @@
     entry_points={
         "console_scripts": [
             "reporting_server=rest_server.__main__:main",
+            "clean_logs=rest_server.clean_logs:main",
```

Comment:
Can we name this to be more specific? `clean_logs` doesn't really imply that it is an auxiliary tool for the reporting server. Maybe add a prefix to the name, have a combined script that expose this function and other future functions as sub commands, or integrate this into the main script with a sub command.

Reply:
I added this `reporting_server_clean_logs` for now. And for the future, if I understand this right, do you want me to create something like: `reporting_server_script --clean_logs`,  `reporting_server_script --migration`, etc?

Reply:
I think it's fine as long as one can easily identify that the script is meant for reporting server. A name like `clean_logs` can be quite vague as to what logs it's cleaning. So either a prefix on all of the reporting server scripts, or a meta-script that combines the functionalities of all the other scripts.